docs/dashboards/README.md

```diff
@@ -0,0 +1,163 @@
+- `k8s.pod.name` (needs to be added to the telemetry data, see the [Kubernetes section](#kubernetes) below)
+
+### Adding `service.instance.id` to the allow list
+`service.name` and `k8s.pod.name` are on the Dynatrace OTLP allow list by default, `service.instance.id` is not. In order to add it follow [this guide](https://docs.dynatrace.com/docs/shortlink/metrics-configuration#allow-list) and add `service.instance.id` to the list.
+
+## Architecture
+Every OpenTelemetry collector has selfmonitoring capabilities, but they need to be activated.
+Self-monitoring data can be exported from the collector via the OTLP protocol.
+The suggested way of exporting selfmonitoring data is to run one collector dedicated for collecting and exporting the selfmonitoring data for the other running collectors, and forwarding that data to Dynatrace.
+Below, you can see a configuration example for a selfmonitoring collector.
+
+```yaml
+# receive selfmonitoring data via gRPC from OTel collector instances.
+receivers:
+  otlp/selfmon:
+    protocols:
+      grpc: 
+        endpoint: 0.0.0.0:4317
+
+processors:
+  # transform cumulative values to deltas. 
+  cumulativetodelta/selfmon: {}
+  # (kubernetes only) retrieves kubernetes attributes for other collectors sending to this collector. See Kubernetes prerequisites below.
+  k8sattributes/selfmon: {}
+
+  # prepend 'sfm.otelcol' to all selfmon metrics - the charts in the {dashboard_name}.json file expects this prefix.
+  transform/selfmon:
+    error_mode: ignore
+    metric_statements:
+      - context: metric
+        statements:
+          - set(name, Concat(["sfm.otelcol", name], "."))
+
+exporters:
+  # Inject DT_ENDPOINT and DT_API_TOKEN as environment variables. This should be the environment where the selfmonitoring data will go.
+  otlphttp/selfmon:
+    endpoint: "${DT_ENDPOINT}/api/v2/otlp"
+    headers:
+      Authorization: "Api-Token ${DT_API_TOKEN}"
+
+  # (optional) logs how many elements were exported to the Dynatrace backend.
+  debug:
+    verbosity: basic
+
+service:
+  # turn on the selfmonitoring for the selfmonitoring collector itself.
+  telemetry:
+    # (kubernetes only) the k8sattributesprocessor does not add attributes for the selfmonitoring collector itself. This is a known limitation of the processor.
+    # These environment variables need to be injected, see the Kubernetes prerequisite section below.
+    resource:
+      k8s.namespace.name: "${env:K8S_POD_NAMESPACE}"
+      k8s.pod.name: "${env:K8S_POD_NAME}"
+      k8s.node.name: "${env:K8S_NODE_NAME}"
+
+    metrics:
+      level: detailed
+      # export data via OTLP OTLP/grpc
+      readers:
+        - periodic:
+            interval: 20000
+            exporter:
+              otlp:
+                # the endpoint of the selfmonitoring collector. In this case, it is assumed that there is a service called `selfmon-collector` that exposes port 4317.
+                endpoint: selfmon-collector:4317
```

Comment:
"selfmon-collector" is only defined in your k8s deployment below, right? Would localhost here as well and also on bare metal deployments?

Reply:
`0.0.0.0` would only work if the collector sends to itself. For all other collectors it does not work. I guess we could do 
```suggestion
                endpoint: <location-of-your-selfmon-collector>:4317
```
or something similar to make it explicit.

Reply:
Alternatively, we could think about distributing the `.yaml` files for Kubernetes along with the json files, but I am not sure if that wouldn't make it even more complicated

Reply:
This particular section here is about the selfmon collector scraping itself specifically, right?

Reply:
This particular section is, but I still opted for using the name of the service as the endpoint instead of `0.0.0.0` so they are consistent. This works, and if someone is copying together yaml files I believe it's easier to have them all be the same to avoid accidentally copying the wrong one.